.gitlab-ci.yml

```diff
@@ -10,23 +10,23 @@
 # at Lawrence Livermore National Laboratory (LLNL).
 #
 # This entire pipeline is LLNL-specific
-##############################################################################
+# #############################################################################
```

Comment:
whitespace

Reply:
I'll merge this now and fix the whitespace in a subsequent PR.